From f71fa3a9548d52dee0a21df2e21badf6efd6f329 Mon Sep 17 00:00:00 2001
From: Scott Kitterman <scott@kitterman.com>
Date: Mon, 10 Mar 2014 19:20:18 -0400
Subject: Change paths in sample conf file to match Debian

Patch-Name: Change-paths-in-sample-conf-file-to-match-Debian.patch
---
 etc/clamav-milter.conf.sample | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/etc/clamav-milter.conf.sample b/etc/clamav-milter.conf.sample
index b7d66a4..ed0d519 100644
--- a/etc/clamav-milter.conf.sample
+++ b/etc/clamav-milter.conf.sample
@@ -64,7 +64,7 @@ Example
 # daemon (main thread).
 #
 # Default: disabled
-#PidFile /var/run/clamav-milter.pid
+#PidFile /var/run/clamav/clamav-milter.pid
 
 # Optional path to the global temporary directory.
 # Default: system specific (usually /tmp or /var/tmp).
@@ -90,7 +90,7 @@ Example
 # with the same socket: clamd servers will be selected in a round-robin fashion.
 #
 # Default: no default
-#ClamdSocket tcp:scanner.mydomain:7357
+ClamdSocket /var/run/clamav/clamd
 
 
 ##
@@ -212,7 +212,7 @@ Example
 # A full path is required.
 #
 # Default: disabled
-#LogFile /tmp/clamav-milter.log
+#LogFile /var/log/clamav/clamav-milter.log
 
 # By default the log file is locked for writing - the lock protects against
 # running clamav-milter multiple times.
